rust-keybroker/keybroker-server/src/verifier.rs

```diff
@@ -73,6 +74,25 @@ pub fn verify_with_veraison_instance(
     // policy also wants to match the RIM value reported by the CCA token with
     // the known-good RIM values supplied on the command line.
     let results = policy::rego_eval(policy, policy_rule, reference_values, &ear_claims)?;
+    if results.to_string() != "true" {
+        // The appraisal failed. In case it's likely due to missing known-good RIM values,
```

Comment:
I think this code is in the wrong place for a couple of reasons. The first reason is that it's CCA-specific, and therefore I'd recommend we put it behind some kind of abstraction so that other contributors can handle the same situation for their architecture. But the second reason ties with my earlier comment. We are conflating the absence of reference values (which is a very specific situation), with all of the other reasons why we might get a non-`true` result from the policy.

If the ref value is missing, we shouldn't eval the policy at all. The logging message needs to say something like "the attestation result could not be evaluated because there is no reference value to use for comparison". But that should be gated on whether the ref value is `Some` vs `None`, rather than trying to gate it on the `true` result from the policy.

Reply:
This has been hopefully addressed with the next patch.